tools/kete.py

```diff
@@ -585,6 +585,7 @@ def run(self):
 
     def json_to_svg(self, js, filename):
         dimensions = js['dimensions']
+        dimensions = [x/10 for x in dimensions]
```

Comment:
honestly, I don't think just dividing by 10 here is enough, it's just adding another magic number. let's check if we can write out the physical dimensions to the SVG and then go based on that.

Reply:
216 is awfully close to 210 mm, so I think we can use this as-is. That's probably where the "Approximately" comes from on the website. Also, unless I'm misreading, the scaling factor to mm is 100, right?

Reply:
Yes, the choose the factor 10 to fit it to format provided by Wacom. But 100 would be the right one for millimeters.

Reply:
How does this behave for the smaller A5 Folio?